### DISCUSSION TITLE
Changing from PHD to Masters cover (Nova-FCT)

### DISCUSSION THREAD

**Discussion by ASilva-fct-UNL** [joaomlourenco/novathesis#463] [Q&A]
Hey, im starting my Masters thesis with the latex template in Overleaf and my colleagues told me that the cover must be blue for the masters thesis and not green (because that one is for the PHD).

I've searched for an answer here but i havent found that yet and I don't know how to change from green to blue.

Thanks in advance

**Comment by joaomlourenco** [comment]
Please read the README in the Project Github site:
https://github.com/joaomlourenco/novathesis/
Then, if you still have questions, please come back.
